PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -823,26 +823,31 @@ private extension PanModalPresentationController {
     func snap(toYPosition yPos: CGFloat, completion: (() -> Void)? = nil) {
         PanModalAnimator.animate(
             { [weak self] in
-                self?.adjust(toYPosition: yPos)
-                self?.isPresentedViewAnimating = true
+                guard let self = self else { return }
+                let didAdjust = self.adjust(toYPosition: yPos)
+                self.isPresentedViewAnimating = didAdjust
```

Comment:
why not just `self.isPresentedViewAnimating = self.adjust(toYPosition: yPos)`?

Reply:
Don't think that it's super obvious what the output of the `adjust` function is.
IMO, `didAdjust` gives it a bit of an additional context